jest.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  moduleNameMapper: {
+    '\\.(css|scss)$': 'identity-obj-proxy',
+    '^actions$': '<rootDir>/src/actions',
+    '^components$': '<rootDir>/src/components',
+    '^reducers$': '<rootDir>/src/reducers',
+    '^containers$': '<rootDir>/src/containers',
+    '^store$': '<rootDir>/src/store'
+  },
+  transform: {
+    '^.+\\.js$': 'babel-jest',
+    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': '<rootDir>/jestConfig/__mocks__/fileTransformer.js'
+  },
+  moduleFileExtensions: [
```

Comment:
these are default module extensions, you dont need to specify it again

Reply:
during our app, we are agreed on using, js and jsx file extension. it's an overkill to allow our file import default to .json, .tsx and ts.
https://jestjs.io/docs/en/configuration#modulefileextensions-array-string